install_config/topics/full_rhv_inventory.adoc

```diff
@@ -4,21 +4,18 @@ Module included in the following assemblies:
 install_config/configuring_rhv.adoc
 ////
 
-= Configuring {product-title} for Red Hat Virtualization 
+= Configuring {product-title} for Red Hat Virtualization
 
-You configure {product-title} for Red Hat Virtualization by modifying the
-Ansible inventory file before you install the cluster.
+You can configure {product-title} for Red Hat Virtualization by modifying the Ansible inventory file before you install the cluster.
 
-. Modify the Ansible inventory file, located at
-*_/etc/ansible/hosts_* by default, to use the following YAML sections:
+. Modify the Ansible inventory file, located at *_/etc/ansible/hosts_* by default, to use the following YAML sections:
 +
 [source,yaml]
 ----
 [OSEv3:children]
 nodes
 masters
 etcd
-glusterfs_registry
```

Comment:
You delete this 'glusterfs_registry'

But I find that  you use the glusterfs as the backend storage: 
```# Registry
openshift_hosted_registry_storage_kind=glusterfs
```

Reply:
@rgolangh @jan-meskal
Roy and Jan, can you answer this?

Reply:
We don't have any Gluster available for that setup, and I don't think Jan tested that with it.
@shalo -  how easy is it to configure the registry post install? if not post install, what is used usually for registry storage, nfs, hostpath? the documentation cover that for deploying on other platforms, like openstack, vmware and so on?

Reply:
The e

Reply:
I can confirm what @rgolangh has said: I haven't done any testing using gluster backend. All the storage backend that I used was NFS. I also don't think I specifically set up `openshift_hosted_registry_storage_kind` at any point.